tests/foreman/ui/test_computeresource.py

```diff
@@ -127,19 +129,25 @@ def test_create_libvirt_resource_different_names_negative(self, name):
         @Assert: A libvirt Compute Resource is not created
 
         """
+        include_list = [' ']
```

Comment:
Do we have particular reason for separate variable? We try to omit variables in our code that has single usage